Websites/bugs.webkit.org/committers-autocomplete.js

```diff
@@ -174,6 +174,9 @@ WebKitCommitters = (function() {
     }
 
     function getIndex(item) {
+        if (!item.classList.contains('committer-suggestion'))
+            item = item.parentNode;
```

Comment:
Can you use `.closest('.committer-suggestion')` ? Seems more future proof.

Reply:
I'm terrible at remembering how to amend a PR with `git-webkit`.